test/blocks/video/video.test.js

```diff
@@ -114,6 +139,88 @@ describe('video uploaded using franklin bot', () => {
     expect(video.hasAttribute('data-play-viewport')).to.be.true;
   });
 
+  it('accessibility controls should pause autoplay videos', async () => {
+    const block = document.querySelector('.video.autoplay.viewportplay');
+    const fetchStub = sinon.stub(window, 'fetch');
+    fetchStub.resolves({
+      total: 19,
+      offset: 0,
+      limit: 19,
+      data: [
+        {
+          key: 'play-motion',
+          value: 'Play',
+        },
+        {
+          key: 'pause-motion',
+          value: 'Pause',
+        },
+        {
+          key: 'play-icon',
+          value: 'play icon',
+        },
+        {
+          key: 'pause-icon',
+          value: 'pause icon',
+        },
+      ],
+      ':type': 'sheet',
+    });
+
+    const a = block.querySelector('a');
+    block.append(a);
+    init(a);
+    const video = block.querySelector('video');
+    decoratePausePlayWrapper(video, '');
+    const pausePlayWrapper = block.querySelector('.pause-play-wrapper');
+    pausePlayWrapper.click();
+    await new Promise((resolve) => { setTimeout(resolve, 1000); });
```

Comment:
A little worried for a timeout of a second, but I understand if it is necessary, just will add time to our entire test suite.